app/models/repository/subversion.rb

```diff
@@ -35,17 +35,44 @@ class Repository::Subversion < Repository
   validates_format_of :url, with: /\A(http|https|svn(\+[^\s:\/\\]+)?|file):\/\/.+\z/i
 
   def self.scm_adapter_class
-    Redmine::Scm::Adapters::SubversionAdapter
+    OpenProject::Scm::Adapters::Subversion
   end
 
-  def self.scm_name
-    'Subversion'
+  def configure(args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `configure(*)` if you want the method to accept any arguments but don't care about them.